microsoft-edge/extensions-chromium/publish/publish-extension.md

```diff
@@ -71,9 +71,15 @@ Go to the [developer dashboard](https://partner.microsoft.com/dashboard/microsof
 <!-- ========================================================================== -->
 ## Step 2:  Upload the extension package
 
-Use the **Packages** webpage to upload the zip file of your extension package.  You can only upload one package at a time.  Your submission is blocked if the package upload isn't successful on the **Packages** webpage.
+1. Go to the **Packages** webpage to upload the zip file of your extension package.
 
-To upload the package, select and drag the package onto the upload field.  Also, you can select **Browse your files**.  After the package is uploaded, your package is validated.  After validation succeeds, review the extension details, and then select **Next** to continue.  If there are validation errors, resolve the issues and try uploading again.
+   You can only upload one package at a time. Your submission is blocked if the package upload isn't successful on the **Packages** webpage.
+
+1. To upload your package, drag and drop the zip file to the **Upload** section of the webpage. Or you can select **Browse your files** to open a dialog box to select the package.
```

Comment:
@ShashankMG123 please check the user interface terms in bold.

Reply:
Not sure if we can call it the Upload section
![image](https://user-images.githubusercontent.com/37444395/157169470-35dc8926-0c8f-4347-8659-d6423bbcbc0d.png)

Reply:
Fixed